charts/logging-opensearch/templates/indextemplate-kernel.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.opensearch.indexTemplates.defaultTemplates.kernel }}
+apiVersion: opensearch.opster.io/v1
+kind: OpensearchIndexTemplate
+metadata:
+  name: {{ include "logging-opensearch.name" . }}-kernel
+  namespace: {{ .Release.Namespace | default "logging" }}
+  labels:
+    {{- include "logging-opensearch.labels" . | nindent 4 }}
+spec:
+  name: kern
```

Comment:
Could you rename that please to "kernel". The two additional characters don't hurt and are more readable.